crates/release_plz_core/src/command/update.rs

```diff
@@ -123,6 +133,27 @@ impl PackagesUpdate {
             })
             .collect()
     }
+
+    pub fn releases(&self) -> Vec<ReleaseInfo> {
+        self.updates
+            .iter()
+            .map(|(package, update)| {
+                let changelog = update.last_changes().unwrap_or(None);
+                let (changelog_title, changelog_notes) = changelog
+                    .map_or(("".to_string(), "".to_string()), |c| {
```

Comment:
Is the empty string a good choice here? I think making those Option and setting them to None may be annoying as missing them would make release-plz fail.

Reply:
I think we should use Option. 
If None, we shouldn't set the fields in the changelog context.

> as missing them would make release-plz fail

Yeah, people need to check if those field are set 👍
Maybe we should update the docs accordingly.